src/site/stages/build/drupal/graphql/nodeQa.graphql.js

```diff
@@ -1,6 +1,7 @@
 const entityElementsFromPages = require('./entityElementsForPages.graphql');
 
 const WYSIWYG = '... wysiwyg';
+const REACT_WIDGET = '... reactWidget';
 const BUTTON = '... button';
 const ALERT_SINGLE = '... alertSingle';
```

Comment:
Can we inline these? Not really sure why they need to be assigned variables 🙂

Reply:
I was following the convention in the rest of the GraphQL files, but yeah, I also think it's ugly